bedrock/firefox/templates/firefox/new/basic/base_platform.html

```diff
@@ -0,0 +1,36 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/new/basic/base_download.html" %}
+
+{% block experiments %}{% endblock %}
+
+{# This template can render with EITHER 'firefox/new/download.ftl' OR 'firefox/new/platform.ftl' active
+ # download.ftl contains strings for the hero and small-links for the platform pages
+ # platform.ftl contains strings for the hero, small-links, and features sections for the platform pages
+ # the view for the platform pages will activate the locale if either language file is active  #}
```

Comment:
If I'm following along correctly, I think the aim here is to activate as many locales as possible for the download pages, correct? I've tested things as thoroughly as I could using your checklist (really helpful!) and things do seem to be in order ✅

That said, understanding this all felt pretty complicated, and I'm still wondering a little if there's a hole in the logic somewhere. There's certainly a level of complexity we're taking on here long term if we decide to go this route. Looking at the number of active locales for each file right now I see:

- platform.ftl = 53 active locales
- download.ftl = 56 active locales
- combined active locales = 61

This is definitely a small win, but I have to ask the question: is this tech debt acknowledgement worth the number of extra locales we gain? What does long term maintenance look like this these pages?

Reply:
There is some number of additional locales we will activate with the deleted strings but it is time consuming to estimate how many. For example: ach, ar, and bn should re-activate 🤞 I found these by looking at the translations in Pontoon. BUT we can get this win without sharing strings between the two ftl files.

There are 8 locales where there are active general download pages but not active platform pages (an, ca, fi, hy-AM, ja, lij, mr, ur). Those are the ones that would benifit from mixing the two language files. (because it turns out we have the strings `Download { -brand-name-firefox } <br>for { -brand-name-windows }` in download.ftl 🤷‍♀️) I have asked Raphael to give me an estimate of the value of activating the platform pages in addition to the general download page.

If the value of having both is low I could separate the HTML back into two templates like we had before but then we are back to having two HTML page to maintain and I feel like the HTML changes more than the strings. 

> What does long term maintenance look like for these pages? How do we plan to simplify this again?

We should never touch platform.ftl or download.ftl again. There is no migrating these languages to a new ftl file or a single ftl file, they have been abandoned. My goal was to make the HTML maintainable and never touch the .ftl files again. 

> Which .ftl file should be used going forward? (fallback strings go both ways currently)

I updated the PR to always use the platform ones on the platform page if they are available. I *think* dev will show the English strings from platform.ftl on the platform page since all files are marked as active. 

> How should localizers understand where to find strings / test their changes?

The basic /new has a URL to trigger it so translators can see those strings. 

The platform ones can be viewed on the platform page. With the update I just made so that the platform ones are always preferred changes to platform.ftl should always be reflected there. Again, I *think* the English will display if there's a missing string and not the fallback. I'll see if I can find a way to check.